schedule/yast/sle/flows/default@svirt-xen-pv.yaml

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
I know that is a mess between the mutiple ways to write it spread in the repo, with `@` with `-` or with `_`, but at least for the default files, let's follow the same pattern, so `default_svirt_xen_pv.yaml`.
When this would be more mature I think we should a ticket to make all naming consistent, but it is fine for now.